app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,10 @@
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:vmSafeMode="true"
+
+        >
```

Comment:
please refract this. This may look a very very small thing but these kind of things doesn't look good in a community project.